api/models/TimeEntry.js

```diff
@@ -23,6 +23,10 @@ module.exports = (sequelize) => {
             allowNull: false,
             unique: true
         },
+        description: {
+            type: DataTypes.STRING,
+            allowNull: false
```

Comment:
@sofiarm21 This should not be a required field right? Also this doesn't match what's in the video since in there it shows Allow Null is true in the database which is the ideal structure I think. Also the TimeEntries that don't come up with a description seem blank in the database in the video when it's probably better if they are left as NULL since we would now kind of have a mix of null/empty states so better to just have NULL instead of an empty string

Reply:
@otech47 I resolved this on the most recent commit, now the descriptions that comes as blank from toggl are stored as NULL.